front/src/components/UI/ActionInput/AddAuction.tsx

```diff
@@ -0,0 +1,74 @@
+import axios from 'axios';
+import { ButtonStyled, FormStyled, LabelStyled, InputStyled, SelectStyled } from './AuctionInput.styled';
+import { isEmpty } from '../../App/SalesArea/SalesArea';
+import { useState } from 'react';
+
+const AddAuction = () => {
+  const [formInputsValidity, setFormInputsValidity] = useState({
+    name: true,
+    productId: true
+  });
+  
+  const addAuctionHandler = (event:any) => {
```

Comment:
Z tymi eventami miałem problem żeby wywalić to any. Nie mogłem znaleźć nigdzie interfejsu lub wbudowanego Eventu, który by to obsługiwał.

Reply:
A skad tutaj bierze sie to elements?